scripts/characters/constructor.gd

```diff
@@ -588,6 +590,40 @@ func build_genealogy(person, mother, father):
 	while percent != 100:
 		percent = build_genealogy_equalize(person, percent)
 	#/ralph9
+	#ralphB - optional consolidation of beastkin/halfkin races on breeding (offspring will have only one beastkin/halfkin type race with total % that would have been split b/n different beastkin/halfkin races)
+	if globals.useRalphsTweaks && globals.expansionsettings.consolidatebeastDNA:
+		#print('Ralph test: consolidatebeastDNA == true')
+		var total_beastkin_race_percent = 0
+		var babys_beastkin_races = []
+		var count_beastkin_races = 0
```

Comment:
`count_beastkin_races` is redundant as `babys_beastkin_races.size()` can provide the same value for less work.

Reply:
comment incorporated for next commit